CMakeLists.txt

```diff
@@ -97,7 +97,7 @@ if (BUILDING_RUNTIME)
     
     if (MSVC)
         add_definitions(/D_CRT_SECURE_NO_WARNINGS /DNOMINMAX)
-        add_compile_options(/wd4267 /wd4251 /FC /utf-8 /W3 /WX)
+        add_compile_options(/wd4267 /wd4251 /wd4244 /FC /utf-8 /W3 /WX)
```

Comment:
Is it necessary?

Reply:
It's not necessary in the future, but since currently caffe.pb.h and caffe.pb.cpp are pre-generated as source code, compile warning is inevitable. When they are in a library later, the compile option will be removed.